tools/pub_get_offline.py

```diff
@@ -71,21 +71,24 @@ def CheckPackage(package):
 
 
 def Main():
-  leading = os.path.join(
-      "src", "third_party", "dart", "tools", "sdks", "dart-sdk", "bin"
-  )
+  script = os.path.realpath(__file__)
```

Comment:
I think the right thing to do is to follow the pattern that we use in other scripts for setting these paths up as globals like in:

https://github.com/flutter/engine/blob/main/tools/githooks/setup.py#L14,

then you can rewrite the paths in `ALL_PACKAGES`.

```python
# Check that this is right.
SRC_ROOT = os.path.dirname(
    os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
)
FLUTTER_DIR = os.path.join(SRC_ROOT, 'flutter')
DART_BIN = os.path.join(SRC_ROOT, 'third_party', 'dart', 'tools', 'sdks', 'dart-sdk', 'dart')

ALL_PACKAGES = [
  os.path.join(FLUTTER_DIR, "ci"),
  ...
]
```